### PR TITLE
N20: Planck ECS

### PR DESCRIPTION
**Pull request by jojolepro**
Hi!

I added Planck ECS to the newsletter. :hearts: 

_Part of #531_

### CODE REVIEW

File: content/posts/newsletter-020/index.md [hunk 1]
```diff
@@ -252,6 +252,37 @@ https://github.com/stevebob/mos6502/blob/master/tetris-hard-drop-patcher/src/mai
...
 
+### [Planck ECS]
+
+![planck logo](planck_ecs.png)
+
+[Planck ECS] ([GitHub][Planck ECS], [Blog][planck_blog],
+[Patreon][planck_patreon]) by [@jojolepro]
+is a brand new minimalist and safe ECS library.
+
+The 1.0 released happened in the past month, featuring: various fixes,
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-020/index.md:263]:
```suggestion
The 1.0 release happened in the past month, featuring: various fixes,
```

**Review by 17cupsofcoffee — APPROVED** [review]
Minor spelling change, but looks good otherwise:

File: content/posts/newsletter-020/index.md [hunk 2]
```diff
@@ -252,6 +252,37 @@ https://github.com/stevebob/mos6502/blob/master/tetris-hard-drop-patcher/src/mai
...
+quality of life improvements, removal of unsafe code and completion of tests and
+documentation.
+
+The library is currently considered completed, which means that all planned
+features are implemented, tested and benchmarked. Future updates will focus on
+performance improvements and usability improvements.
+
+[Planck ECS] is also used in [Shotcaller][planck_shotcaller] which is featured in this newsletter
+too.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-020/index.md:272]: Line length is tripping up the linter:

```suggestion
[Planck ECS] is also used in [Shotcaller][planck_shotcaller] which is featured in
this newsletter too.
```

**Review by ozkriff — APPROVED** [review]
:+1:

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**